cabby/data/metagraph/utils.py

```diff
@@ -55,6 +55,12 @@ def convert_pandas_df_to_metagraph(
       nx.set_node_attributes(g, attribute_dict, metadata_name)
   return g
 
+def get_osmid_from_wd_relations_row(row: pd.Series):
+  osmid = hash(''.join(list(row.values)))
+  if osmid < 0:
+    osmid *= -1
```

Comment:
What is going on here? A brief example would probably help make it clearer.

Reply:
Hopefully fixed by 8cb04ea4046158db624b1350a62c500a84503ef6

Let me know if the docstring doesn't clarify!

Reply:
Ah yes, it does, thanks! In the code, can you call it `fake_osmid` or `pseudo_osmid`?

Side note: I'm a bit worried about the fact that we now have osmid's that aren't osmid's in the mix -- seems like a a possible gotcha down the line. Fine to keep it as is for now, but worth keeping in mind.